apps/nextjs-website/src/lib/header.ts

```diff
@@ -50,7 +50,10 @@ export const makeHeaderProps = (
     href: '/',
   },
   ...(header.data.attributes.ctaButtons && {
-    ctaButtons: header.data.attributes.ctaButtons,
+    ctaButtons: header.data.attributes.ctaButtons.map((ctaBtn) => ({
+      ...ctaBtn,
+      color: header.data.attributes.theme === 'dark' ? 'negative' : 'primary',
+    })),
```

Comment:
The color should be defined by the theme. I would omit this override. If currently the theme is being ignored, let's open a ticket to address it.

```suggestion
    ctaButtons: header.data.attributes.ctaButtons,
```